### ISSUE TITLE
Add Tor support Feature Request

### ISSUE THREAD

**Issue by kaschmirrr** [nextcloud/android#7866]
Hello, 
it would be really cool if tor support could be added in the app so someone can access his onion nextcloud server without the need to always start the external orbot app in addition.

**Comment by AndyScherzinger** [comment]
@kaschmirrr I guess the request is a ducplicate of #606, no?

**Comment by kaschmirrr** [comment]
Not exactly, i think of direct integration of the tor service in the app so yoi dont have to use the orbot app in addition. If you need a good example for what i mean have a look at the phoenix wallet app, which has tor directly integrated.

**Comment by kaschmirrr** [comment]
Here is an article why this topic is important and nextcloud could benefit very much by tor integration: https://www.expressvpn.com/blog/apps-integrate-tor-dark-web-encryption/

**Comment by kn0wmad** [comment]
We are adding Nextcloud to our platform, with default remote access via Tor.  Our users would very much love to see this feature, is there anything we can do to help with integration?

**Comment by andreasmjg** [comment]
I would also like to see this. Accessing Nextcloud instance at my home, from my mobile without Tor, is like broadcasting to everyone exactly where I live. :/
